x-pack/plugins/enterprise_search/public/applications/enterprise_search_overview/components/elasticsearch_client_instructions/elasticsearch_dotnet.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import dedent from 'dedent';
+
+import { EuiCodeBlock, EuiLink, EuiText, EuiSpacer } from '@elastic/eui';
+
+export const ElasticsearchDotnet: React.FC = () => {
+  return (
+    <>
+      <EuiText>
+        <p>Elastic builds and maintains two .NET clients for Elasticsearch:</p>
+        <ul>
+          <li>
+            Elasticsearch.Net, a low level, dependency free client that has no opinions about how
+            you build and represent your requests and responses.
+          </li>
+          <li>
+            NEST, a high level client that maps all requests and responses as types, and comes with
+            a strongly typed query DSL that maps 1 to 1 with the Elasticsearch query DSL.
+          </li>
+        </ul>
+        <EuiLink
+          target="_blank"
+          href="https://www.elastic.co/guide/en/elasticsearch/client/net-api/current/introduction.html"
```

Comment:
We should use the docsLink [service](https://github.com/elastic/kibana/blob/main/packages/kbn-doc-links/src/get_doc_links.ts#L604) for these URLs. It will future-proof these in case the URLs change. This goes for all links in this PR that start with `https://www.elastic.co/guide/en/`.